scala-api/src/main/scala/com/cloudera/livy/package.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.livy
+
+package object client {
```

Comment:
This is a little weird, because all classes are in `com.cloudera.livy`, but the user has to import `com.cloudera.livy.client._` to get the implicit conversion.

Perhaps all classes should be in the same module, and it should be a child module of `com.cloudera.livy`? Just don't call that module `scala` because Scala's relative import rule will break things.